src/services/link/index.js

```diff
@@ -115,16 +119,20 @@ module.exports = function () {
 
 				if (email) {
 					// generate import hash
-					await hashService.create({
-						toHash: email,
-						save: true,
-						patchUser: true,
-					}).then((generatedHash) => {
-						linkInfo.hash = generatedHash;
-					}).catch((err) => {
-						logger.warn(err);
-						return Promise.resolve('Success!');
-					});
+					const user = (await app.service('users').find({ query: { email: data.toHash } })).data[0];
```

Comment:
more less this duplicates the changes above

Reply:
yes, but the overlap is so small (especially since errors are handled differently), that I wouldnt pull it out right now if thats fine with you.